src/status_im/chat/handlers/input.cljs

```diff
@@ -388,7 +388,8 @@
               new-sel      (->> command-args
                                 (take (+ 3 arg-pos))
                                 (input-model/join-command-args)
-                                (count))
+                                (count)
```

Comment:
Very minor style point, but parenthesis around `(count)` here are actually superflous, the same can be written as:
```clj
(->> command-args
     (take (+ 3 arg-pos))
     (input-model/join-command-args)
     count
     (min (count input-text)))
```